docs/src/pages/index.js

```diff
@@ -28,6 +106,39 @@ function HomepageHeader() {
   );
 }
 
+function Framework({ image, title, description }) {
+  return (
+    <div className={clsx("col col--4")}>
+      <div className="text--center padding--md">
+        <img
+          alt="logo"
```

Comment:
How do you feel about adding the framework title in alt text, for e.g.: "Panel logo", maybe something like:

```js
alt={`${title} logo`}
```

Reply:
Yes, good catch, thanks!